spec/controllers/articles_controller_spec.rb

```diff
@@ -145,4 +145,26 @@
       end
     end
   end
+
+  describe 'GET #show' do
+    context 'when user is logged in' do
+      before(:each) do
+        allow(controller).to receive(:current_user).and_return(user)
+      end
+
+      it 'returns a 200 OK HTTP response' do
```

Comment:
Might be good to add a test that verifies it's displaying the right information as well.

Reply:
Great idea. I was going to leave that out as the next story will change the tests but after thinking about it I realise that each story should be independent of all others and should have comprehensive testing - even if those tests are expected to change.

Reply:
That's right! I like to see things as this: consider that every code you push to production will potentially live unmodified forever. You never know what the future holds, and that improvement you have in mind will never get prioritized. Thus, make sure your code is the best shape possible.

Reply:
It's little learning curves like this that I'm so excited to be experiencing before jumping into sprint work! Acceleration project FTW! 🙌